snyk/snyk-sbt-plugin
# fix: remove injected sbt script only if it exists

#### What does this PR do?

The library was missing a check if the sbt script was injected successfully. In case it wasn't it failed to remove it because of a missing cleanup callback.

This patch set refactors the code to handle that case properly.

#### Where should the reviewer start?

In `pluginInspect`, line 135.

#### How should this be manually tested?

Make the `try` block in `pluginInspect` (starting at line 136) error out. The plugin should handle the failure gracefully.

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?

N/A

#### Screenshots

N/A

#### Additional questions

N/A
